### DISCUSSION TITLE
Allows Users To Highlight Selected Areas With Desired Colors.

### DISCUSSION THREAD

**Discussion by QuangLanh** [clientIO/joint#2401] [Q&A]
### Introduction

I want users to be able to mark a portion of the diagram according to their preferences after drawing. They can achieve this by creating a frame, similar to the one in Paint, and selecting the desired color to annotate each section of the diagram they have drawn.
Please tell me how to do the above feature.
Thanks.

### Steps to reproduce

![image](https://github.com/clientIO/joint/assets/91791283/577047ee-a1ec-4ee9-9ee7-0747545548c5)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by QuangLanh** [comment]
please reply me early !
i need demo my project next week.
Thanks.

**Comment by kumilingus** [comment]
If you want your users to be able to draw a rectangle, here's an example: https://www.jointjs.com/demos/shapes-drawing
You should set the rectangle `z` attribute low so it displays below the other elements, or set the `fill` color to `none`.
